src/server.c

```diff
@@ -5582,7 +5584,9 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                                              aof_bio_fsync_status == C_OK) ? "ok" : "err",
             "aof_last_cow_size:%zu\r\n", server.stat_aof_cow_bytes,
             "module_fork_in_progress:%d\r\n", server.child_type == CHILD_TYPE_MODULE,
-            "module_fork_last_cow_size:%zu\r\n", server.stat_module_cow_bytes));
+            "module_fork_last_cow_size:%zu\r\n", server.stat_module_cow_bytes,
+            "total_startup_load_deleted_keys:%lld\r\n", server.stat_persistence_startup_load_keys_deleted,
+            "total_startup_load_unowned_slots:%lld\r\n", server.stat_persistence_startup_load_unowned_slots));
```

Comment:
Let's just log these, I don't think we need to add a server field for these, since they will be static for the lifetime of the instance.